test/parallel/test-quic-client-handshake-with-test-server.mjs

```diff
@@ -0,0 +1,60 @@
+// Flags: --experimental-quic
+import { hasQuic, isAIX, isWindows, skip, getPort } from '../common/index.mjs';
```

Comment:
CI is failing with
```console
common.PORT cannot be used in a parallelized test
```

Line of code which throws error:
https://github.com/nodejs/node/blob/c6316f9db9869864cea84e5f07585fa08e3e06d2/test/common/index.js#L1031-L1036

The `common.PORT` is returned by getPort function, and it's also used in two other tests in this PR.